docs/features/tokens/Tokens.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { TokenTable, textRenderer, fontSizeRenderer, backgroundRenderer, borderRenderer, iconRenderer, lineHeightRenderer, boxShadowRenderer, radiiRenderer, sizingRenderer, spacingRenderer, fontWeightRenderer } from "./TokenTable";
+import { Meta } from "@storybook/addon-docs";
+import { Div } from "@components/html";
+import { ShareGateTheme,  ThemeProvider } from "@components/styling"
+import { ColorPaletteTable } from "./ColorPaletteTable";
+import { Table } from "@stories/components";
+
+<Meta
+    title="Features/Tokens"
+    id="tokens"
+/>
+
+# Tokens
+
+Tokens are Orbit's reusable values, providing a common language between developers and designers. These can be used as `CSS` variables or [Style Props](/docs/style-props--page) values.
```

Comment:
I think you could rework a little bit this text to move "These can be used as `CSS` variables or [Style Props](/docs/style-props--page) values" in the usage section since this is describing the usage.